src/types_core.ts

```diff
@@ -24,7 +24,7 @@ export class CoreConfig {
     const REQUIRED_LANG = process.env.REQUIRED_LANG || "en"
     const PROJECT_ID = process.env.PROJECT_ID
     const CLIENT_EMAIL = process.env.CLIENT_EMAIL
-    const PRIVATE_KEY = process.env.PRIVATE_KEY
+    const PRIVATE_KEY = process.env.PRIVATE_KEY.replace(/\\n/g, "\n") // prevents dotenv escaping
```

Comment:
strange, I don't need this to have the bot work properly

Reply:
Might be something with versions? idk

Won't break yours though